docs/docker.rst

```diff
@@ -0,0 +1,79 @@
+.. _eprosima_docker_image:
+
+eProsima Docker Image
+=====================
+
+eProsima provides the eProsima XRCE-DDS Suite Docker image for those who want a quick demonstration of XRCE-DDS running on an Ubuntu
+platform. It can be downloaded from `eProsima's downloads page <https://eprosima.com/index.php/downloads-all>`_.
+
+This Docker image was built for Ubuntu 20.04 (Focal Fossa).
```

Comment:
I don't see why is this relevant

Reply:
https://github.com/eProsima/Fast-DDS-docs/blame/master/docs/docker/includes/intro.rst#L4